spec/cc/workspace/path_tree_spec.rb

```diff
@@ -58,7 +69,7 @@ class CC::Workspace
       within_temp_dir do
         make_fixture_tree
 
-        tree = PathTree.new(".")
+        tree = PathTree.for_path(".")
         tree.exclude_paths(["code/does-not-exist"])
         expect(tree.all_paths.sort).to eq [".git/", "code/a/", "code/foo.rb", "foo.txt", "lib/"]
```

Comment:
This might be outside scope of this PR, but maybe we could consider in the future blowing up here instead? Not sure which is better.